### PR TITLE
Initial Migration Draft to wipe the network gracefully

### PR DESCRIPTION
**Pull request by stefanbenten**
Ties in to V3-1440. It resets the network gracefully by removing all data from the storagenodes.

## Code Review Checklist
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

Thanks for submitting a PR!

### CODE REVIEW

File: pkg/piecestore/psserver/psdb/psdb.go [hunk 1]
```diff
@@ -205,6 +205,13 @@ func (db *DB) Migration() *migrate.Migration {
 					`CREATE INDEX IF NOT EXISTS idx_bwa_serial ON bandwidth_agreements (serial_num)`,
 				},
 			},
+			{
+				Description: "Initiate Network reset",
+				Version:     4,
+				Action: migrate.SQL{
+					`UPDATE ttl SET expires = '1553641200' WHERE created <= '1553641200' `,
```
Comment (jtolds) [inline pkg/piecestore/psserver/psdb/psdb.go]: can we have created be something in april? expires is fine. also aren't these numbers and not strings? we shouldn't have the quotes, right?

File: satellite/satellitedb/migrate.go [hunk 2]
```diff
@@ -362,6 +362,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				Description: "Initiate Network reset",
+				Version:     9,
+				Action: migrate.SQL{
+					`DELETE FROM PATHDATA`,
```
Comment (jtolds) [inline satellite/satellitedb/migrate.go]: is this the only table we want to clear? okay

File: pkg/piecestore/psserver/psdb/psdb.go (hunk 1 above)
Comment (stefanbenten) [reply pkg/piecestore/psserver/psdb/psdb.go]: I was expecting of SQL handling the cast, but i will double check 👍

File: satellite/satellitedb/migrate.go (hunk 2 above)
Comment (stefanbenten) [reply satellite/satellitedb/migrate.go]: If all pointers are gone, we should be fine afaik.
All other services are depending on them and i didnt want to remove any billing information.

Main issue is, that i cannot run that query on the unit tests, as they arent aware of the table..

**Pull request merged**